Source/WebKitLegacy/mac/Configurations/WebKitLegacy.xcconfig

```diff
@@ -124,9 +122,17 @@ TAPI_USE_SRCROOT = $(TAPI_USE_SRCROOT$(WK_XCODE_15));
 TAPI_USE_SRCROOT_XCODE_SINCE_15 = YES;
 TAPI_VERIFY_MODE[sdk=iphone*] = Pedantic;
 
+// FIXME: Replace -extra-*-header and -exclude-*-header flags with build setting equivalents once rdar://99885053 is completed in all needed toolchains.
```

Comment:
Is this the right radar? Looking at its title, I don't understand the relationship.